ceph-releases/ALL/debian/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -0,0 +1,16 @@
+# add the necessary repos, referring to the repo lists in ERP18.06
+    apt-key add /tmp/ceph-release.asc && \
+    echo "deb http://obs.linaro.org/ERP:/ceph-ansible/Debian_9 ./" \
+      >> /etc/apt/sources.list.d/erp.list && \
+    echo "deb http://obs.linaro.org/ERP:/17.12/Debian_9 ./" \
+      >> /etc/apt/sources.list.d/erp.list && \
+    echo "deb http://obs.linaro.org/ERP:/18.06/Debian_9/ ./" \
+      >> /etc/apt/sources.list.d/erp.list && \
+    echo "deb http://deb.debian.org/debian/ __OS_CODENAME__-backports main" \
+      >> /etc/apt/sources.list.d/erp.list && \
+    echo "deb http://obs.linaro.org/ERP:/staging/Debian_9/ ./" \
```

Comment:
Is the staging tree stable enough not to break builds are every run ?

Reply:
Drop too. Not needed.

Reply:
agree